compiler/src/iree/compiler/Codegen/LLVMGPU/test/reduction_pipeline_transform.mlir

```diff
@@ -27,37 +27,35 @@ hal.executable.variant public @cuda_nvptx_fb, target = <"cuda", "cuda-nvptx-fb",
   }
 }
 }
+
 //   CHECK-LABEL: func.func @group_reduction
 //     CHECK-DAG:   %[[C0:.*]] = arith.constant 0 : index
-//     CHECK-DAG:   %[[C1:.*]] = arith.constant 1 : index
-//     CHECK-DAG:   %[[F0:.*]] = arith.constant dense<0.000000e+00> : vector<f32>
 //     CHECK-DAG:   %[[workgroup_id_x:.*]] = hal.interface.workgroup.id[0] : index
-//     CHECK-DAG:   %[[SHMEM_ALLOC:.*]] = memref.alloc() {alignment = 128 : i64} : memref<1x2xf32, 3>
+//     CHECK-DAG:   %[[SHMEM_ALLOC:.*]] = memref.alloc() {alignment = 128 : i64} : memref<1x64xf32, 3>
 //     CHECK-DAG:   %[[TIDX:.]] = gpu.thread_id  x
-//     CHECK-DAG:   %[[TIDY:.]] = gpu.thread_id  y
```

Comment:
This is still expected to further evolve.

Not sure how to reduce churn on tests except by isolating, separating out and only testing the production of shuffle + e2e correctness for the fully assembled thing.

Reply:
If we don't need to change tests when changing the implementation, we have poor test coverage.